grafana/grafana
# Loki: Add dashboard and panel names as headers

**What is this feature?**

This PR adds dashboard and panel titles as headers to outgoing requests to Loki. Currently behind the `lokiSendDashboardPanelNames` feature flag.

How to test:
1. Enable the `lokiSendDashboardPanelNames` feature flag
2. Set up a Loki datasource: `make devenv loki`
3. Create a proxy in between Loki and Grafana: `mitmproxy --mode=reverse:http://localhost:3100 --listen-host=0.0.0.0 --listen-port=9998 --set console_mouse=false --ssl-insecure`
4. Create a new Loki data source in Grafana targeting http://localhost9998
5. Do a query in Grafana. Check the request headers in `mitmproxy` and you should see a `X-Dashboard-Title` and `X-Panel-Title` header.
